tests/test_watson.py

```diff
@@ -353,6 +353,9 @@ def test_save_current_without_tags(watson):
             assert isinstance(result['start'], (int, float))
             assert result['tags'] == []
 
+            dump_args = json_mock.call_args[1]
+            assert dump_args['ensure_ascii'] is False
```

Comment:
@k4nar @willdurand Is it ok to test calling params only? I don't figure out how to check the dumped json value.